ledger/compact_merkle_tree.py

```diff
@@ -261,10 +261,24 @@ def leafCount(self) -> int:
     def nodeCount(self) -> int:
         return self.hashStore.nodeCount
 
-    def verifyConsistency(self, expectedLeafCount = -1) -> bool:
-        if expectedLeafCount > 0 and expectedLeafCount != self.leafCount:
+    @staticmethod
+    def get_expected_node_count(leaf_count):
```

Comment:
Wouldn't it be better to move it to MerkleTree or some utils class? 

Even if we are not going to have other implementations we already have class hierarchy for this, so we either should follow it or drop it. MerkleTree is used as a type of tree argument, but then we explicitly use CompactMerkleTree, this is confusing.